### DISCUSSION TITLE
'Build ipa for distribution' exited with status code "65"

### DISCUSSION THREAD

**Discussion by Juliakits** [codemagic-ci-cd/codemagic-docs#2275] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

iOS (Swift, Objective-C)

### Steps to reproduce

<img width="824" alt="Снимок экрана 2023-12-18 в 12 26 13" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/153610652/a1eab40b-f1b9-437c-b808-bd8a9003a940">

### Expected results

It was expected that the ipa file would be as going to be.

### Actual results

gives an error: 'Build ipa for distribution' exited with status code "65" and it's not clear what the problem is

### Build id (optional)

65800faecd6a135b33a1ffa9

**Comment by icarusdust** [comment, marked as answer]
Hi @Juliakits, first off, you can add ` /tmp/xcodebuild_logs/*.log` to the `artifacts` section in YAML which will give you a zip file containing all the necessary logs including the error messages.
As for this particular issue, you are getting the following error message:

`......../Config.xcconfig:1:1: error: unable to open configuration settings file `

From this point on, it should be straightforward to solve it. You need to check what is wrong with the given directory, is`Config.xcconfig` in place, whether its content is corrupt or not etc.

**Reply by Juliakits** [reply]
@icarusdust that work perfectly, Thanks for the support!

**Reply by btoo** [reply]
thank you @icarusdust for the support and thank you @Juliakits for reporting this!
